src/localization/modalStrings.js

```diff
@@ -20,7 +20,8 @@ export const modalStrings = new LocalizedStrings({
     edit_the_requisition_comment: 'Edit the requisition comment',
     edit_their_reference: 'Edit their reference',
     finalise_customer_invoice: 'Finalise will lock this invoice permanently.',
-    finalise_requisition: 'Finalise will send this requisition and lock it permanently.',
+    finalise_supplier_requisition: 'Finalise will send this requisition and lock it permanently.',
+    finalise_customer_requisition: 'Finalise will generate a finalised customer invoice, adjust inventory, and lock this requisition permanently.',
```

Comment:
Is this up to date with this implementation? The customer invoice already exists yes? To the user it doesn't exist sure.

Speaking of which, is it possible to just hold off assigning a serial number until the related requisition is finalised?